esti/system_test.go

```diff
@@ -73,6 +73,14 @@ func setupTest(t *testing.T) (context.Context, logging.Logger, string) {
 	return ctx, logger, repo
 }
 
+func tearDownTest(t *testing.T, repoName string) {
+	ctx := context.Background()
+	name := makeRepositoryName(t.Name())
+	logger := logger.WithField("testName", name)
+	deleteRepository(ctx, t, repoName)
```

Comment:
It's important to keep in mind that deleting the repository, the data uploaded to the bucket will still exist. Ultimately if you use the same lakeFS instance for testing over a long period - this will spam and bloat your bucket